appsdk/configurable.go

```diff
@@ -423,6 +424,22 @@ func (dynamic AppFunctionsSDKConfigurable) SetOutputData() appcontext.AppFunctio
 	return transform.SetOutputData
 }
 
+// SetOutputDataWithContentType sets the output data (with content type) to that passed in from the previous function.
+// It will return an error and stop the pipeline if data passed in is not of type []byte, string or json.Mashaler
+// This function is a configuration function and returns a function pointer.
+// TODO: rework this for v2 to collapse to one function (above)
+func (dynamic AppFunctionsSDKConfigurable) SetOutputDataWithContentType(parameters map[string]string) appcontext.AppFunction {
```

Comment:
After some thought on this, we don't need this extra function. We can add the optional parameter to `SetOutputData` w/o it breaking backward compatibility.  These configure functions are only used internally by our reflection code.

Reply:
Ok, I'll remove it, configure the original function, and then test again.

Reply:
Code resolved in latest commit.